Solutions/DNS Essentials/Analytic Rules/DNSRequestToKnownDomainIOCs.yaml

```diff
@@ -105,7 +105,7 @@ eventGroupingSettings:
   aggregationKind: AlertPerResult
 alertDetailsOverride:
   alertDisplayNameFormat: "'{{Name}}' was detected. From client source: '{{SrcIpAddr}}'"
-  alertDescriptionFormat: "Alert Description: '{{Description}}'"
+  alertDescriptionFormat: "Alert Description: '{{Description}}'\n\nClient source '{(SrcIpAddr)}' was detected as making request to '{(DnsQuery)}'"
```

Comment:
I think the word "source" is not needed, here and in most if not all other cases you used it in.

Reply:
thanks. done